android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -598,6 +601,16 @@ private void checkOrCreateChannel(NotificationManager manager) {
         channel.enableLights(true);
         channel.enableVibration(true);
 
+        if (soundUri != null) {
+            AudioAttributes audioAttributes = new AudioAttributes.Builder()
+                    .setContentType(AudioAttributes.CONTENT_TYPE_SONIFICATION)
+                    .setUsage(AudioAttributes.USAGE_ALARM)
+                    .build();
+            channel.setSound(soundUri, audioAttributes);
+        } else {
+            channel.setSound(null, null);
```

Comment:
What is the purpose of this line ?
Reset the sound if there is not passed in parameter ?